### PR TITLE
feat:#50 Cache issues when a new version is released

### PR DESCRIPTION
**Pull request by austininseoul**
https://github.com/PhiloBiblon/philobiblon-ui/issues/50

### CODE REVIEW

File: frontend/nuxt.config.js [hunk 1]
```diff
@@ -77,6 +78,22 @@ export default {
     icon: false,
     workbox: {
       // Configure your service worker options here
+      cacheAssets: false,
```
Comment (jmformenti) [inline frontend/nuxt.config.js:81]: I feel that all this config lines could be a bit risky from a perfomance and introduce maybe some not desired side-effect.
I propose to not include this part, at least initally, and keep the other changes.
What do you think?

File: frontend/nuxt.config.js [hunk 2]
```diff
@@ -148,5 +165,10 @@ export default {
 
   // Build Configuration: https://go.nuxtjs.dev/config-build
   build: {
+    filenames: {
```
Comment (jmformenti) [inline frontend/nuxt.config.js:168]: Nice idea!

File: frontend/plugins/version-check.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export default function ({$config}) {
```
Comment (jmformenti) [inline frontend/plugins/version-check.js:1]: Nice idea!

File: frontend/nuxt.config.js (hunk 1 above)
Comment (austininseoul) [reply frontend/nuxt.config.js:81]: yea we don't have to include

**Pull request merged**